src/journal/journald-server.c

```diff
@@ -2503,7 +2503,9 @@ int server_init(Server *s, const char *namespace) {
                 r = server_open_audit(s);
                 if (r < 0)
                         return r;
-        }
+                log_info("Audit messages are collected.");
+        } else
+                log_info("Audit messages are ignored.");
```

Comment:
"ignored" sounds like we'd see them but drop them. That's not the case. Use a stronger word, "disabled" or so.